python/external_config/external_command.py

```diff
@@ -326,6 +326,80 @@ def execute(self, pre_cache=False):
         :raises: :class:`RuntimeError` on execution failure.
         :returns: Output from execution session.
         """
+        return self._execute(pre_cache)
+
+    def execute_on_multiple_entities(self, pre_cache=False, entity_ids=None):
+        """
+        Executes the external command in a separate process. This method
```

Comment:
I like this approach a lot!! Thanks for trying it out!

In order to avoid the redundancy, perhaps we could just shorten the docstrings for `execute_on_multiple_entities` and `_execute` and have them refer back to `execute` for the details?

Reply:
👍